### PR TITLE
get all ratings for professor functionality added

### PR DESCRIPTION
**Pull request by aeller-design**
(* Ignore the last pull request to main....wrong branch *)

-added both function and api request for get all ratings by professor
-added a unit test
-modified one of the professors in the test file to have reviews

### CODE REVIEW

File: backend/models/professor-services.js [hunk 1]
```diff
@@ -82,6 +82,25 @@ async function professorRatingUpdate(prof, newRating) {
...
+// get all reviews for a professor - return null if professor does not exist
+async function getAllReviews(professor) {
+  try {
+    const professorModel = Connection.getConnection().model(
+      "Professor",
+      ProfessorSchema
+    );
+    const professorToFind = await professorModel.findOne({ name: professor });
+    if (professorToFind === null) {
+      return professorToFind;
```
Comment (sleepyStick) [inline backend/models/professor-services.js:94]: is there a test case that expects this return statement? (aka testing the null case)

**Comment by sleepyStick** [comment]
aside from the comment above, it looks good to me 👍

File: backend/models/professor-services.js (hunk 1 above)
Comment (aeller-design) [reply backend/models/professor-services.js:94]: great catch!  I will add one.

**Review by wes-curl — APPROVED** [review]
This looks good. Can't wait to use it :)

**Comment by ashahrod** [comment]
Looks good to me!

**Review by sleepyStick — APPROVED** [review]

**Review by ashahrod — APPROVED** [review]

**Review by sajanjohal — APPROVED** [review]
Look good to me. Happy to see you added test cases as well.

**Pull request merged**